src/notifications/getting_started.md

```diff
@@ -0,0 +1,77 @@
+# Implementing mobile notifications
+
+The Breez SDK - *Liquid* Notification Plugin provides developers a simple solution to improve the payment experience on a mobile device. No longer does the application need to be in foreground when receiving payments. When the Notification Plugin is added to process push notifications, the application can be in the background or even closed.
+
+## How it works
+
+The process involves using a Notification Delivery Service (NDS) acting as an intermediary host by the application developer. The NDS must provide a public facing webhook URL where a POST request can be sent to when a notification needs to be delivered to the application. The NDS then forwards the data sent in the webhook POST request via push notification to the application. When the application then receives the push notification, the SDK Notification Plugin can be used to process the event.
+
+![Breez SDK - Liquid Notifications](../images/BreezSDK_Liquid_Notifications.png)
+
+### Push notification data
+
+When receiving POST request data on the webhook URL, the NDS should get the `token`, `platform` and optional `app_data` from the URL params. If necessary, convert the token to a token that can be sent to the push notification service. Then format the data to be sent to the push notification service, which includes converting any JSON data into a string format. 
+
+By default the Notification Plugin should receive the push notification data in the following format:
+
+```json
+{
+    "notification_type": "swap_updated",
+    "notification_payload": "{ \"id\": \"\", \"status\": \"\" }",
+    "app_data": ""
+}
+```
+The structure and fields of this data can be changed by [customising the push messages](custom_messages.md) handling in the Notification Plugin to reflect how your NDS sends this data over push notifications.
+
+## Use cases
+
+The Notification Plugin handles several use cases by default to automatically process push notifications sent via the NDS from when an SDK service calls the registered webhook. If your use case isn't covered by the Notification Plugin, you can extend the plugin to [handle custom notifications](custom_notifications.md).
+
+#### Updating a swap
+
+When making a payment via lightning or a Bitcoin onchain address, the SDK uses a swap service to convert between the Liquid sidechain and Bitcoin onchain or lightning. A swap service is used to monitor changes to the swap status and send push notifications to wake the device to progress the swap. The Notification Plugin when receiving this notification from the NDS will start the SDK and let the SDK complete the swap or progress it to the next state. 
```

Comment:
```suggestion
When making a payment via lightning or a Bitcoin onchain address, the SDK uses a swap service to convert between the Liquid sidechain and Bitcoin onchain or Lightning. A swap service is used to monitor changes to the swap status and send push notifications to wake the device to progress the swap. When the Notification Plugin receives this notification from the NDS, it will start the SDK and let the SDK complete the swap or progress it to the next state. 
```